src/components/Clusters/components/oidc-params.ts

```diff
@@ -12,7 +12,16 @@ export function parseOIDCparams({ exec: commandData }: KubeconfigOIDCAuth) {
   let output: any = {};
 
   commandData.args.forEach(arg => {
-    const [argKey, argValue] = arg.split('=');
+    const regex = new RegExp('^(?<key>[^=\\s]+)(?:=(?<value>.*$))');
```

Comment:
Could we replace the regex with something simpler?

Reply:
Do you want me to make the regex simpler or do you want me to **not** use regex at all?

Reply:
Honestly speaking, it is already simplified and leaves out some aspects of the real world you'd normally have to cover. Don't forget there's normally a shell environment in between and thus you could also expect to receive param values like this:

```bash
--param-without-value
--param-with-value=123

# everything below here is not supported yet
--param-with-value value-separated-by-space-instead-of-comma
--param-with-value="123"
--param-with-value='123'
--param-with-value=$(cat ...)
--param-with-value=`cat ...`
```

Reply:
I though about removing the regex at all.

Reply:
You mean something like this? 

```ts
const parts = arg.split('=');
const argKey = parts[0];
const argValue = parts.slice(1).join('=');
```

From the bottom of my heart, please don't go this way.

Reply:
@dbadura @mrCherry97 Can we move forward here? For the reasons mentioned above, I can't recommend to go with a different approach than regex.

Reply:
@v0lkc I'm not a fan of the regexes like this, because it will be painful to debug in the future then there will be some improvements or bugs. In this case, I don't see any better solution, so please provide some comment which is describing this regex